src/components/app.tsx

```diff
@@ -116,10 +116,11 @@ class App extends React.Component<Props & { match: any; location: any }> {
   }
 
   public render() {
+    const showExample = this.props.showExample ? true : false;
```

Comment:
Why?

Reply:
I just added it for better understanding, we can remove it not needed.

Reply:
How does this help understanding to add a ternary that just copies the variable?

Reply:
Okay, yeah that doesnt make much sense, I am removing it

Reply:
You can keep the temp variable if you want.